src/main/java/com/cinema/model/MovieSession.java

```diff
@@ -0,0 +1,59 @@
+package com.cinema.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private Movie movie;
+    @ManyToOne(cascade = CascadeType.PERSIST)
```

Comment:
I think that thisk task must be solved with join fetch.You need use FetchType Lazy (At Many to One relations we have Eager by default).And not sure about your CascadeType,in my opinion on this step is redundant

Reply:
I don't agree with you, we have no conditions in our  TO DO tasks  to use JOIN FETCH, and EAGER fetch type here is appropriate, because we make only one additional object, and it is not so bad for performance in our case. As for cascade type-you are right, it's redundant. Removed